src/pages/[...slug].js

```diff
@@ -0,0 +1,188 @@
+import PropTypes from "prop-types";
```

Comment:
Shouldn't this page be in `src/pages/stories/[...slug].js`?

Reply:
@kilemensi  we had that conversation [here](https://github.com/CodeForAfrica/PesaYetu/pull/61#discussion_r687165639)